dali/plugin/plugin_manager.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
```

Comment:
I know, we're copypasting this, but maybe good time to fix it ;)
It's 2018th, 2017 is redundant in this header :)